ui.frontend/src/utils.js

```diff
@@ -310,12 +311,28 @@ class Utils {
                     _path,
                     _element: elements[i]
                 });
-                const event = new CustomEvent(Constants.FORM_CONTAINER_INITIALISED, { "detail": formContainer });
+                const event = new CustomEvent(Constants.FORM_CONTAINER_INITIALISED, {"detail": formContainer});
                 document.dispatchEvent(event);
             }
         }
     }
 
+    static async registerCustomFunctionsByUrl(url) {
+        if (url != null && url.trim().length > 0) {
+            const customFunctionModule = await import(/*webpackIgnore: true*/ url);
+            const keys = Object.keys(customFunctionModule);
+            for (let i = 0; i < keys.length; i++) {
+                const name = keys[i];
+                const funcDef = customFunctionModule[keys[i]];
+                if (typeof funcDef === 'function') {
+                    const functions = [];
+                    functions[name] = funcDef;
+                    FunctionRuntime.registerFunctions(functions);
```

Comment:
this call to register functions should be outside for loop.

Reply:
done